qa/integration-tests/src/test/java/io/camunda/zeebe/it/health/BrokerMonitoringEndpointTest.java

```diff
@@ -43,20 +43,20 @@ static void setUpClass() {
   void shouldGetReadyStatus() {
     await("Ready Status")
         .atMost(60, TimeUnit.SECONDS)
-        .until(() -> given().spec(brokerServerSpec).when().get("ready").statusCode() == 204);
+        .until(() -> given().spec(brokerServerSpec).when().get("ready").statusCode() == 200);
```

Comment:
:question: This test works because, it is automatically redirected to the new path `actuator/health/readiness`. Is that right?

Reply:
Exactly, the status code here is the final one.